server/eb/app.py

```diff
@@ -60,6 +60,7 @@ def _before_adding_routes(app, app_config):
         script_hashes, style_hashes = WSGIServer.get_csp_hashes(app, app_config)
         csp = {
             "default-src": ["'self'"],
+            "connect-src": ["'self'"],
```

Comment:
why is this needed?  My understanding is that if is unspecified, it falls back to the `default-src` definition, which is identical.   @nacin - do you know?

Reply:
@bkmartinjr when the client code pings our sentry hostname, the console reported it looking in `connect-src` first. I added this so as to be as specific as possible about the scope of that hostname. Happy to change it, if there is a better solution.